README.md

```diff
@@ -1,3 +1,75 @@
 # Ansible role: Helm
 
 This role handles the creation and updates of Helm charts.
+
+[![Build Status](https://travis-ci.com/Flaconi/ansible-role-helm.svg?branch=master)](https://travis-ci.com/Flaconi/ansible-role-helm)
+[![Version](https://img.shields.io/github/tag/Flaconi/ansible-role-helm.svg)](https://github.com/Flaconi/ansible-role-helm/tags)
+
+## Requirements
+
+* Ansible 2.5
+
+
+## Additional variables
+
+Additional variables that can be used (either as `host_vars`/`group_vars` or via command line args):
+
+| Variable                     | Description              |
+|------------------------------|--------------------------|
+| `helm_version`               | Helm version to download |
```

Comment:
With the new changes introduced, the description should now be changed to something like: `Exact locally expected Helm version (defaults to: 2.12.3)`

In case we notice that this gets to annoying, we can address it in the future to make this an optional variable.

Reply:
Done!